aws/asg.go

```diff
@@ -85,6 +87,45 @@ func getAutoScalingGroupsByName(service autoscalingiface.AutoScalingAPI, autoSca
 	return result, nil
 }
 
+func getOwnedAutoScalingGroups(service autoscalingiface.AutoScalingAPI) (map[string]*autoScalingGroupDetails, error) {
+	params := &autoscaling.DescribeAutoScalingGroupsInput{}
+	resp, err := service.DescribeAutoScalingGroups(params)
```

Comment:
I think we need to use: https://docs.aws.amazon.com/sdk-for-go/api/service/autoscaling/#AutoScaling.DescribeAutoScalingGroupsPages instead because `DescribeAutoscalingGroups` will only list a limited number of autoscaling groups (The limit is `100`, but still we should handle this edge case).

Reply:
aha, will fix